### PR TITLE
Fix initial brightness value

### LINKED ISSUE #3084: Brightness resets on first gesture brightness change after playback activity open
Thia bug persists since a long time, I didn't come around reporting it.

Stock Android 7.1.1
XT1771

### PR DESCRIPTION
**Pull request by vhsw**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
Fixes  #3084

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java [hunk 1]
```diff
@@ -909,6 +909,14 @@ private void setInitialGestureValues() {
                 final float currentVolumeNormalized = (float) getAudioReactor().getVolume() / getAudioReactor().getMaxVolume();
                 volumeProgressBar.setProgress((int) (volumeProgressBar.getMax() * currentVolumeNormalized));
             }
+
+            final float screenBrightness =  getWindow().getAttributes().screenBrightness;
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java]:
```suggestion
            final float screenBrightness = getWindow().getAttributes().screenBrightness;
```

Comment (vhsw) [reply app/src/main/java/org/schabi/newpipe/player/MainVideoPlayer.java]: Fixed that

**Comment by Stypox** [comment]
Could you provide a test apk?

**Comment by vhsw** [comment]
Here is the test apk

[app-debug.zip](https://github.com/TeamNewPipe/NewPipe/files/4224961/app-debug.zip)

**Comment by Stypox** [comment]
I tested the app and it works as expected. Thank you @vhsw ! :-D
@TobiGr I am merging this even though it is not in the milestone since it is a simple change

**Comment by TobiGr** [comment]
@Stypox If you have time, can you please create a RC and create a PR + Issue? I'd like to release a new version as soon as possible, but the time I can spend on NewPipe is quite limited the next two weeks.

**Comment by wb9688** [comment]
@TobiGr: I'd wait until yt_new is finished. Tomorrow I probably have another 4 hours and them I'm hopefully much more productive.

**Comment by Stypox** [comment]
I don't have the keys to sign a release RC, so I'm just going to build a Debug RC apk, is that ok?
I assume the way to build RCs is exactly the same as the way to build normal debug apks, isnt't?
The PR has to be created between the dev and the master branch, hasn't it?
Also, would #3098 be postponed to 0.18.5?

@wb9688 Ok, even though it could have some problems at the beginning since it is not thoroughly tested. But we will test using RCs, so that's ok. Thank you :-)

**Comment by TobiGr** [comment]
@Stypox For the RC, I use my debug key, too. I'd wait for mauricio's review of #3098. If he does not answer until Friday, I assume it is good to merge and will be included in the 0.18.4. 

Regarding the new YouTube version: we should release it asap, but not in 0.18.4. We still have 11 days and can release 0.18.5 as soon as the extractor is stable.

**Comment by Stypox** [comment]
I will create the PR&issue later today or tomorrow (I have to study ;-) )

**Pull request merged**